src/els_parser.erl

```diff
@@ -73,15 +75,15 @@ find_attribute_pois(Form, Tokens) ->
           %% The first atom is the attribute name, so we skip it.
           [_|Atoms] = [T || {atom, _, _} = T <- Tokens],
           ExportEntries =
-            [ els_poi:new(Pos, export_entry, {F, A})
+            [ poi(Pos, export_entry, {F, A})
```

Comment:
The `els_poi:new/3` function takes a `location`. I believe it should take a range, instead. For now (to avoid a big refactoring) I'm simply adding a wrapper function, but this should be cleaned up. Also, the `els_range` module does not really make sense outside of the `els_poi` and the two modules could be merged.